r124/inref/ydb312gtm8182d.m

```diff
@@ -0,0 +1,30 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+
+gtm8182
+	DO getPaths
+
+	SET $ZGBLDIR=INST1gbldir
+	SET x=$get(^a)
+	; ZSYSTEM "$ydb_dist/mupip replic -source -start -log=source.log -buf=1 -instsecondary=DUMMY -passive -updok"
+	ZSYSTEM "$MSR STARTSRC INST1 INST2"
+	SET ^a=1
```

Comment:
Display that we are about to do an update after starting the source server in the same M program. Also indicate that this used to SIG-11 before #312 fixes.